Update policy-csp-networklistmanager.md (#10196)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/policy-csp-networklistmanager.md

<!--/Scope-->
<!--Description-->
This policy setting provides the list of URLs (separated by Unicode character 0xF000) to endpoints accessible only within an enterprise's network. If any of the URLs can be resolved over HTTPS, the network would be considered authenticated.  

> When entering a list of TLS endpoints in Microsoft Endpoint Manager, you should follow this format, even in the UI:  

Author: ```suggestion
When entering a list of TLS endpoints in Microsoft Endpoint Manager, you must follow this format, even in the UI:  
```

<!--/Scope-->
<!--Description-->
This policy setting provides the list of URLs (separated by Unicode character 0xF000) to endpoints accessible only within an enterprise's network. If any of the URLs can be resolved over HTTPS, the network would be considered authenticated.  

> When entering a list of TLS Endpoints in MEM (Microsoft Endpoint Management), you must follow this format even in the UI:  

Reviewer: ```suggestion

When entering a list of TLS endpoints in Microsoft Endpoint Manager, you should follow this format, even in the UI:  
```

```<![CDATA[https://nls.corp.contoso.com&#xF000;https://nls.corp.fabricam.com]]>```  

> - The HTTPS endpoint must not have any additional authentication checks such as login or multi-factor authentication.

Reviewer: ```suggestion
- The HTTPS endpoint must not have any additional authentication checks, such as login or multifactor authentication.
```

- The HTTPS endpoint must be an internal address not accessible from outside the corporate network.

> - The client must trust the server certificate, so the CA cert the HTTPS server cert chains to must be present in the client machines root certificate store.

Reviewer: ```suggestion
- The client must trust the server certificate, so the CA certificate that the HTTPS server certificate chains to must be present in the client machine's root certificate store.
```

- A certificate should not be a public certificate.

> 
> 

Reviewer: ```suggestion
```
